_data/simple-icons.json

```
@@ -14259,6 +14259,12 @@
             "hex": "FD5750",
             "source": "https://serverless.com"
         },
+        {
+            "title": "ServiceNow",
+            "hex": "6AC63F",
```

It seems that the original logo color has changed.

```suggestion
            "hex": "62D84E",
```